### ISSUE TITLE
Remove maintainer_email in twisted/*/topfiles/setup.py

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#3482]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)| @thijstriemstra reported|
|-|-|
|Trac ID|trac#3482|
|Type|task|
|Created|2008-10-21 12:52:45Z|
|Branch|https://github.com/twisted/twisted/tree/maintainer-email-3482|

There are still references to maintainer email addresses in the twisted/*/topfiles/setup.py files, see [twisted/web/topfiles/setup.py](twisted/web/topfiles/setup.py) for example. I suggest completely removing those ```maintainer_email``` lines.

<details><summary>Searchable metadata</summary>

```
trac-id__3482 3482
type__task task
reporter__thijs thijs
priority__normal normal
milestone__ 
branch__branches_maintainer_email_3482 branches/maintainer-email-3482
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__ 
time__1224593565000000 1224593565000000
changetime__1225057034000000 1225057034000000
version__None None
owner__ 
cc__exarkun
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_1"></a>|@thijstriemstra commented|
|-|-|

(In [25033]) Branching to 'maintainer-email-3482'

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_2"></a>|@thijstriemstra removed owner|
|-|-|

Applied fixes in r25035, up for review.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_3"></a>|@exarkun set owner to @radix|
|-|-|

The changes to version numbers in flow and pair should be reverted.  They're smaller because new releases weren't done for them in 8.1.

I think the release manager should take a look at this.

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_4"></a>|@thijstriemstra commented|
|-|-|

Reverted those version nr changes in r25098, assigning to radix for review.

**Comment by twisted-trac** [comment]
|[<img alt="radix's avatar" src="https://avatars.githubusercontent.com/u/227068?s=50" width="50" height="50">](https://github.com/radix)<a name="note_5"></a>|@radix set owner to @thijstriemstra|
|-|-|

Looks good, please merge.

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_6"></a>|@thijstriemstra set status to `closed`|
|-|-|

(In [25120]) Merge maintainer-email-3482: Get rid of maintainer_email metadata to eliminate spam and encourage usage of the mailinglist.

Author: thijs
Reviewer: exarkun, radix
Fixes: [#3482](https://github.com/twisted/twisted/issues/3482)

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
